data/countries_names.txt

```diff
@@ -49289,7 +49262,6 @@
   ru = Мец
   ar = متز
   cs = Mety
-  fr = Metz, Thionville
```

Comment:
Did you check that the missing fr translation is correctly taken from the English one on the device?

Reply:
Yes